etl/steps/data/garden/demography/2023-02-03/life_expectancy.py

```diff
@@ -387,6 +387,41 @@ def add_americas(frame: pd.DataFrame) -> pd.DataFrame:
     return df
 
 
```

Comment:
I didn't review the details here (because you mostly move the code), but it would be good to use `paths` in the future to avoid multiple updates of versions in this file

Reply:
I've moved the recipe for this step to `dag/archive/demography.yml` for now, so it should not be re-baked anytime soon. 

But yeah, looking at this code now feels like... 😱